app/assets/javascripts/application.js

```diff
@@ -112,6 +112,11 @@ function onContentLoad(){
 
   var tz = jstz.determine();
   $.cookie('timezone', tz.name(), { path: '/' });
+
+  // Removes all the templates from forms as they can't be cleared by strong parameters.
+  $('form').on('submit', function(e) {
```

Comment:
just include it in the same place we have the lookup_keys js?

On Tue, May 5, 2015 at 2:47 PM, Daniel Lobato García <
notifications@github.com> wrote:

> In app/assets/javascripts/application.js
> <https://github.com/theforeman/foreman/pull/2055#discussion_r29661548>:
>
> > @@ -112,6 +112,11 @@ function onContentLoad(){
> >
> >    var tz = jstz.determine();
> >    $.cookie('timezone', tz.name(), { path: '/' });
> > +
> > +  // Removes all the templates from forms as they can't be cleared by strong parameters.
> > +  $('form').on('submit', function(e) {
>
> This only happens when editing Host interfaces, and Lookup keys right? Can
> it be moved there instead? Seems like a waste of time to do it everywhere.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/theforeman/foreman/pull/2055/files#r29661548>.
>